includes/wpmenucart-woocommerce.php

```diff
@@ -8,8 +8,9 @@ public function __construct() {
 		}
 	
 		public function menu_item() {
-			// make sure cart is loaded! https://wordpress.org/support/topic/activation-breaks-customise?replies=10#post-7908988
-			$this->maybe_load_cart();
+			$this->maybe_load_session();  // in backend (full site editor) the wc session is null
+			$this->maybe_load_customer();
```

Comment:
I think we should limit these calls to the backend, not touching anything if it's not absolutely necessary.